distribution/docs/src/main/resources/_contents/_configuring/configuring-intro-contents.adoc

```diff
@@ -0,0 +1,18 @@
+
+.Configuration Requirements
+[NOTE]
+====
+Because components can easily be installed and uninstalled, it's important to remember that for proper ${branding} functionality, at least the Catalog API, one Endpoint, and one Catalog Framework implementation must be active.
+====
+
+${branding} can be configured in several ways, depending on need:
+
+[NOTE]
+====
+While there are multiple ways to configure ${branding} for use, the recommended method is to use the ${admin-console}.
+====
+
+* RECOMMENDED: Using the browser and the ${admin-console}. <<_configuring_${branding-lowercase}_from_the_admin_console, Configuring ${branding} From the ${admin-console}>>
```

Comment:
Not sure if the problem is with the Mac Previewer I'm using to look read the PDF but those section links don't work for me.